lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -22,40 +31,77 @@
 
 required.add_argument('-r', '--run_number', help="Run number if the flat-field data",
                       type=int, required=True)
-required.add_argument('-p', '--pedestal_run', help="Run number of the drs4 pedestal run",
-                      type=int, required=True)
-optional.add_argument('-v', '--version', help="Version of the production",
-                      type=int, default=0)
+optional.add_argument('-p', '--pedestal_run', help="Pedestal run to be used. If None, it looks for the pedestal run of the date of the FF data.",
+                      type=none_or_str)
+
+version=lstchain.__version__.rsplit('.post',1)[0]
```

Comment:
What Max was proposing is to keep the `.post...` part explicitly.

Reply:
Yes, please just use `__version__` so that test / development tests cannot be confused with the normal productions.

Reply:
Sorry, but this is exactly what I do not want, because it demands to recreate a new calibration tree at each commit. 
I think this solution is good both for the official tree  than for the debugging (head) code

Reply:
@FrancaCassol made me realize that it would be very difficult in the debugging phase because one should reproduce a new tree subdirectory at each commit.

Reply:
Then I would propose to just remove the commit part, not the post part.

Reply:
what does the post number mean?

Reply:
I think it is the commit hash.

Reply:
So I do not like to have it in the tree either

Reply:
The part after `post` is the number of commits since the tag and then comes `git<hash>`.

I was proposing to keep the `post` part, not the part after that. So that any development version is e.g.
```
0.7.5.post
```

Note, that with the switch  to `setuptools_scm` in #737, this would be:

```
0.7.6.dev
```

Reply:
I think it is an unuseful complication for debugging. It does not concerns the official tree (which will be alway with a release), and developer people do not need to add a "post" to the personal tree to remember that they are working with a development release. Also it could be I like to use part of the official tree but if my tree has a different version name I can not do it.

Reply:
I'm fine with the version line as it is